agent/src/api/v1/proxy.rs

```diff
@@ -22,35 +23,40 @@ pub async fn relay_request(
     service_name: &str,
     source_instance_id: &Uuid,
 ) -> Result<Response, ServalError> {
-    let node_info = discover_service(service_name).await.map_err(|err| {
-        log::warn!("proxy_unavailable_services failed to find a node offering the service; service={service_name}; err={err:?}");
+    let mesh = MESH.get().expect("Peer network not initialized!");
```

Comment:
(Thinking aloud 🤔 ) Is it worth promoting this to `ServalError`, or (since this happening would be a Programming Error™) is the catastrophic nature of the `.expect` preferable?